src/components/FamilyManifestModal/FamilyManifestModal.js

```diff
@@ -150,10 +62,8 @@ const enhance = compose(
   withState('isDisabled', 'setIsDisabled', false),
 );
 
-const FamilyManifestModal = ({
```

Comment:
just personal preference, not sure if you agree but I kinda like the component named variables